ligo/errors.mligo

```diff
@@ -149,6 +149,9 @@
 (* Some problem in cumulative seconds per liquidity computation. *)
 [@inline] let internal_spl_cumulative_err = 326n
 
+(* Flip of `seconds_per_liquidity_outside` failed. (This is an invariant of the contract). *)
+[@inline] let flip_seconds_per_liquidity_outside_err = 327n
```

Comment:
shouldn't this be an internal error, aka only happen due to a bug in the contract?

If so, I guess we should keep the convention to include `internal` in its name, I think

Reply:
Aye good point! I just copied `flip_fee_growth_outside_err` and did not notice that, I'll rename both errors then.

Reply:
Done